### PR TITLE
solution

### PR DESCRIPTION
**Pull request by deelray**
[DEMO LINK](https://deelray.github.io/react_add-todo-form/)

### CODE REVIEW

**Review by ira-cherpak — APPROVED** [review]
Good work 🌟

File: src/App.js [hunk 1]
```diff
@@ -1,19 +1,128 @@
...
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <h1>Add todo</h1>
+        <form
+          action="./api/todo.js"
+          method="POST"
+          onSubmit={this.submitHandler}
```
Comment (aionova) [inline src/App.js:75]:
```suggestion
          onSubmit={this.handleSubmit}
```

**Review by aionova — CHANGES_REQUESTED** [review]
Good start!

**Review by aionova — APPROVED** [review]
Good job!

**Pull request closed without merging**